services/authorizerService.js

```diff
@@ -0,0 +1,31 @@
+/*
+Authorizer services
+*/
+import Authorizer from "../models/authorizerModel.js";
+
+const getRequestStatusId = async (id) => {
+    try {
+        const idRequest = await Authorizer.getStatusId(id);
+        console.log("ID a comprobar:", idRequest);
+        if (!idRequest) {
+            return null;
+        }
+        else {
+            if (idRequest == 1)
+                return 2;
+            else if (idRequest == 2)
+                return 3;
```

Comment:
After reviewing with @andresjaramillo7 and @101-Coconsulting/api this should be the proper flow.
```suggestion
            if (idRequest == 2)
                return 3;
            else if (idRequest == 3)
                return 4;
```